website/config/routes.js

```diff
@@ -510,9 +511,10 @@ module.exports.routes = {
   // Redirects for external links from the Fleet UI & CLI, including to fleetdm.com and to external websites not
   // maintained by Fleet. These help avoid broken links by reducing surface area of links to maintain in the UI.
   'GET /learn-more-about/chromeos-updates': 'https://support.google.com/chrome/a/answer/6220366',
-  'GET /learn-more-about/just-in-time-provisioning': '/docs/deploy/single-sign-on-sso#just-in-time-jit-user-provisioning',
-  'GET /learn-more-about/custom-os-settings': '/docs/using-fleet/mdm-custom-os-settings',
+  'GET /learn-more-about/just-in-time-provisioning': 'https://fleetdm.com/docs/deploy/single-sign-on-sso#just-in-time-jit-user-provisioning',
```

Comment:
Looks like this needs the domain removed again (I added a new link in a different PR that was merged, so guessing this change got undone resolving a merge conflict).
```suggestion
  'GET /learn-more-about/just-in-time-provisioning': '/docs/deploy/single-sign-on-sso#just-in-time-jit-user-provisioning',
```